core/src/test/java/org/apache/iceberg/TestBase.java

```diff
@@ -108,6 +108,14 @@ public class TestBase {
           .withPartitionPath("data_bucket=1") // easy way to set partition data for now
           .withRecordCount(1)
           .build();
+  static final DeleteFile FILE_B2_DELETES =
```

Comment:
I think a better name would be `FILE_B_EQ_DELETES` because I think that's how you're using it.